src/ripple/protocol/impl/Issue.cpp

```diff
@@ -36,6 +40,67 @@ to_string(Issue const& ac)
     return to_string(ac.account) + "/" + to_string(ac.currency);
 }
 
+Json::Value
+to_json(Issue const& is)
+{
+    Json::Value jv;
+    jv[jss::currency] = to_string(is.currency);
+    if (!isXRP(is.currency))
+        jv[jss::issuer] = toBase58(is.account);
+    return jv;
+}
+
+Issue
+issueFromJson(Json::Value const& v)
+{
+    if (!v.isObject())
+    {
+        Throw<std::runtime_error>(
+            "issueFromJson can only be specified with a 'object' or string "
+            "Json value");
```

Comment:
I think this exception message is wrong. It shouldn't include "or string". I don't see a way that `v` could be a string, since the next line accesses object fields.